Tests/OpenCombineTests/Helpers/TestDecodable.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/22/19.
+//
+import Foundation
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+struct TestDecodable: Codable, Equatable {
```

Comment:
We probably don't need a new type just to test encoding/decoding, we can go with a dictionary `[String : String]` or an array `[Int]`.